plugins/PostProcessingPlugin/scripts/MaxVolumetricSpeed.py

```diff
@@ -0,0 +1,275 @@
+        enable_volumetric_t0 = self.getSettingValueByKey("enable_volumetric_t0")
+        enable_volumetric_t1 = self.getSettingValueByKey("enable_volumetric_t1") if extruder_count > 1 else False
+        filament_dia_t0 = float(extruder[0].getProperty("material_diameter", "value"))
+        filament_dia_t1 = filament_dia_t0
+        if enable_volumetric_t0:
+            retract_enabled_t0 = bool(extruder[0].getProperty("retraction_enable", "value"))
+            cura_retract_speed_t0 = float(extruder[0].getProperty("retraction_retract_speed", "value"))
+            cura_prime_speed_t0 = float(extruder[0].getProperty("retraction_prime_speed", "value"))
+            speed_e_reset_t0 = cura_retract_speed_t0 if cura_retract_speed_t0 >= cura_prime_speed_t0 else cura_prime_speed_t0
+            max_E_flow_rate_t0 = self.getSettingValueByKey("max_E_flow_rate_t0")
+            enable_jerk_adjustment_t0 = self.getSettingValueByKey("enable_jerk_adjustment_t0")
+            max_e_jerk_t0 = round(self.getSettingValueByKey("max_e_jerk_t0"),1) if enable_jerk_adjustment_t0 else global_stack.getProperty("machine_max_jerk_e", "value")
+            default_jerk_t0 = global_stack.getProperty("machine_max_jerk_e", "value")
+        else:
+            cura_retract_speed_t0 = float(extruder[0].getProperty("retraction_retract_speed", "value"))
+            cura_prime_speed_t0 = float(extruder[0].getProperty("retraction_prime_speed", "value"))
+            speed_e_reset_t0 = cura_retract_speed_t0 if cura_retract_speed_t0 >= cura_prime_speed_t0 else cura_prime_speed_t0
+
+        if enable_volumetric_t1 and extruder_count > 1:
+            retract_enabled_t1 = bool(extruder[1].getProperty("retraction_enable", "value"))
+            cura_retract_speed_t1 = float(extruder[1].getProperty("retraction_retract_speed", "value"))
+            cura_prime_speed_t1 = float(extruder[1].getProperty("retraction_prime_speed", "value"))
+            speed_e_reset_t1 = cura_retract_speed_t1 if cura_retract_speed_t1 >= cura_prime_speed_t1 else cura_prime_speed_t1
+            filament_dia_t1 = float(extruder[1].getProperty("material_diameter", "value"))
+            max_E_flow_rate_t1 = self.getSettingValueByKey("max_E_flow_rate_t1")
+            enable_jerk_adjustment_t1 = self.getSettingValueByKey("enable_jerk_adjustment_t1")
+            max_e_jerk_t1 = round(self.getSettingValueByKey("max_e_jerk_t1"),1) if enable_jerk_adjustment_t1 else global_stack.getProperty("machine_max_jerk_e", "value")
+            default_jerk_t1 = global_stack.getProperty("machine_max_jerk_e", "value")
+        elif not enable_volumetric_t1 and extruder_count > 1:
+            cura_retract_speed_t1 = float(extruder[1].getProperty("retraction_retract_speed", "value"))
+            cura_prime_speed_t1 = float(extruder[1].getProperty("retraction_prime_speed", "value"))
+            speed_e_reset_t1 = cura_retract_speed_t1 if cura_retract_speed_t1 >= cura_prime_speed_t1 else cura_prime_speed_t1
```

Comment:
Could you find a way to merge those 2 ifs ? They are quite big and very error-prone